### ISSUE TITLE
I believe there is an issue with Game Log files

### ISSUE THREAD

**Issue by DanR89** [zhaofengli/explorer#1]
Have tried a few different Ingress data visualisation options now and all fail, I believe Niantic have changed the data they are sending to customers so It can not be processed any more.

**Comment by zhaofengli** [comment]
Yeah, I suppose that's the case. I'm no longer playing Ingress and it takes a long time for Niantic to send me a new dump (at least that was my experience - maybe they are faster now). Do you have some snippets of the new files that you can share? You can change the coordinates/names to dummy values, but just keep the format in tact.
